### DISCUSSION TITLE
Themes I Made for Musikcube

### DISCUSSION THREAD

**Discussion by iwsssnow** [clangen/musikcube#660] [Show and tell]
Nordic

![nordic](https://github.com/clangen/musikcube/assets/157258655/2ea457aa-a633-4aaf-b371-c48e4d1c2e9b)
![nordic_command-bar-off_transparent-bg](https://github.com/clangen/musikcube/assets/157258655/53429b53-fbcf-42f3-81f8-a1ded33edb94)

[Nordic.json](https://github.com/clangen/musikcube/files/13999674/Nordic.json)

Red

![red](https://github.com/clangen/musikcube/assets/157258655/f01f6e97-11de-4e17-aba2-fe0914dda144)
![red_command-bar-off_transparent-bg](https://github.com/clangen/musikcube/assets/157258655/b663d05d-c1cd-4835-9d82-02c7d9d1f5ae)

[Red.json](https://github.com/clangen/musikcube/files/13999675/Red.json)

Gray

![gray](https://github.com/clangen/musikcube/assets/157258655/bd5f426b-45bd-4c99-a6ad-a76021e5426b)
![gray_command-bar-off_transparent-bg](https://github.com/clangen/musikcube/assets/157258655/cdae0d57-5999-4dde-9781-851eff06e51a)

[Gray.json](https://github.com/clangen/musikcube/files/13999676/Gray.json)

**Comment by mounta11n** [comment]
very nice!

**Reply by iwsssnow** [reply]
Cheers!

**Comment by iwsssnow** [comment]
Green

![Green-010](https://github.com/clangen/musikcube/assets/157258655/e4f0ccac-4a72-4756-bf17-a3af42192f90)
![Green-020](https://github.com/clangen/musikcube/assets/157258655/6af5b712-518f-4936-abd1-f2a7a0f66c7c)

[Green.json](https://github.com/user-attachments/files/15522923/Green.json)

**Comment by iwsssnow** [comment]
I made a few minor color changes to the Nodic theme above.

Nordic II

![nord-II-20](https://github.com/clangen/musikcube/assets/157258655/a607ddd3-4050-4f3b-b50c-4335a511a3f8)
![nord-II-10](https://github.com/clangen/musikcube/assets/157258655/7c7f7380-a4cd-4932-903a-3b30d6db97c7)

[Nordic-II.json](https://github.com/user-attachments/files/15878739/Nordic-II.json)

**Comment by nesticle8bit** [comment]
I have a newbie question, how can I install them?

**Reply by iwsssnow** [reply]
Hi nesticle8bit.

Go to the musikcube config folder and make a new folder called "themes". Put the .json file in there.

![json-themes-folder](https://github.com/clangen/musikcube/assets/157258655/8aad8349-f604-4733-a3f1-f820e3a997d3)

**Reply by nesticle8bit** [reply]
Thank you so much! ❤️

**Comment by iwsssnow** [comment]
Yellow

![yellow-010](https://github.com/clangen/musikcube/assets/157258655/887d08de-90f9-4c98-92d8-4e22f1a6ac49)
![yellow-020](https://github.com/clangen/musikcube/assets/157258655/f3f2f4d5-5064-49e3-8d12-01b75873e48a)
![yellow-030](https://github.com/clangen/musikcube/assets/157258655/80047889-6e4e-4d85-9ca4-409c96b0e7a0)
[Yellow.json](https://github.com/user-attachments/files/16154835/Yellow.json)


Hello Kitty

![000000-cube-hk-010](https://github.com/clangen/musikcube/assets/157258655/512cf25b-4d75-4887-a341-0db45a15b3f2)
![000000-cube-hk-020](https://github.com/clangen/musikcube/assets/157258655/223e3263-256f-44e0-a79e-5cd78abc35cf)
![000000-cube-hk-030](https://github.com/clangen/musikcube/assets/157258655/7e0bf456-ae84-4585-8e97-3e7a631935a9)
[Hello-Kitty.json](https://github.com/user-attachments/files/16154853/Hello-Kitty.json)

![hello-kitty-breaking-bad](https://github.com/clangen/musikcube/assets/157258655/9ca0732f-1a21-449a-8804-bf4c04ef1486)

**Reply by hidroquino** [reply]
Sorry for the question, how you put the kello kitty background ? 
I'm using the terminal of windows, and even with transparency I cannot put any background at all.

https://apps.microsoft.com/detail/9n0dx20hk701?hl=es-es&gl=ES

**Reply by iwsssnow** [reply]
Hi hidroquino.

The black/pink background you see in my post is just a desktop wallpaper I made to highlight the colors of the musikcube theme. I used that background only for the purpose of taking the screenshot, then cropped the shot to the image you see above. Here is a full desktop example of my current desktop:

![mcube-dtshot](https://github.com/user-attachments/assets/1f1f8f87-6699-4a95-886b-bdd1e38891c0)

So, there isn't a black/pink background image specifically for the theme. I know some terminals have a feature that will let you add a background image. If yours has that feature, I can make one for you, just tell me the dimensions you need.

**Reply by iwsssnow** [reply]
I just did a test adding a background image on my xfce4 terminal. It works. The test background image size is 1024x768. That's probably good for any terminal running musikcube. 

![cube-bg-test](https://github.com/user-attachments/assets/3e855834-79eb-4519-bdf4-6bb40b2259ce)

**Reply by hidroquino** [reply]
@iwsssnow thanks you for your response, I'm on windows and I'm unable to do the transparency with that terminal. 
I'll try on Linux then, thank you so much for your help.

**Reply by emgee-909** [reply]
You can do transparency effect in Windows terminal. My Musikcube is running on local Debian server
<img width="1927" height="1027" alt="musikcube" src="https://github.com/user-attachments/assets/081898f9-2795-424d-9b9c-2f3df6d6d5e4" />

**Comment by iwsssnow** [comment]
Purple

![purple-010](https://github.com/clangen/musikcube/assets/157258655/0a0a01f7-f553-423b-86b7-1420312ed283)
![purple-020](https://github.com/clangen/musikcube/assets/157258655/c9f051fb-4aa8-4da5-a7ac-8f6890402b9d)
![purple-030](https://github.com/clangen/musikcube/assets/157258655/79e6d6a9-35f8-4292-89bc-5a8148a91b1f)
[Purple.json](https://github.com/user-attachments/files/16165284/Purple.json)
